content/2022/en/LCNC-SEC-05-Security-Misconfiguration.md

```diff
@@ -11,6 +11,8 @@ title: "LCNC-SEC-05: Security Misconfiguration"
 | --- | --- | --- | --- |
 | 3 | 2 | 3 | 3 |
 
+In building a solution, many choices have to be made that can impact security. This risk is a bit of a “catch-all” where a user makes choices to settings that create risk. They can range from how/where sensitive data is stored, encryption settings, and more. 
```

Comment:
I'd like to offer a slightly different perspective: Platforms offer a wide variety of features, each with its own balance of productivity vs risk. It's up to users to make the right choices when combining those features.

Reply:
Thank you! Rewrite: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/pull/56/commits/a780d25b03dc5c814bbd9295c8ec4c74aaa96ef5